packages/functional-tests/tests/postVerify/syncForcePasswordChange.spec.ts

```diff
@@ -2,33 +2,31 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-import { expect, test } from '../../lib/fixtures/standard';
-let email;
-const password = 'password';
+import { expect, test, password } from '../../lib/fixtures/standard';
+
 const newPassword = 'new_password';
```

Comment:
nit: We can delete this line and use the `import`ed value from the standard fixtures file.

Reply:
incorporated. Thanks!